buildlib/azure-pipelines.yml

```diff
@@ -0,0 +1,35 @@
+# See https://aka.ms/yaml
+
+trigger:
```

Comment:
let's add glob for release branches as well like in ucx

Reply:
Done